features/checkout/addressing_order/choosing_billing_and_shipping_address_from_address_book.feature

```diff
@@ -0,0 +1,34 @@
+@checkout
+Feature: Choosing an address from address book
+    In order to address an order by choosing it from my address book
+    As a customer
+    I want to be able to quick fill in my address information
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store ships everywhere for free
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And I am a logged in customer
+        And I have an address "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States", "Arkansas" in my address book
+
+    @ui @javascript
+    Scenario: Choosing shipping address from address book
+        Given I have product "PHP T-Shirt" in the cart
+        And I am at the checkout addressing step
+        When I choose "Seaside Fwy" street for shipping address
+        Then I should have this address filled as shipping address
```

Comment:
Then address "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States", "Arkansas" should be filled as shipping address